tools/sklearn/main_macros.xml

```diff
@@ -212,8 +212,23 @@ def get_cv(literal):
 
   <token name="@GET_SCORING_FUNCTION@">
 def get_scoring(scorer):
+  import warnings
+  def balanced_accuracy_score(y_true, y_pred):
+      C = metrics.confusion_matrix(y_true, y_pred)
+      with np.errstate(divide='ignore', invalid='ignore'):
+          per_class = np.diag(C) / C.sum(axis=1)
+      if np.any(np.isnan(per_class)):
+          warnings.warn('y_pred contains classes not in y_true')
+          per_class = per_class[~np.isnan(per_class)]
+      score = np.mean(per_class)
+      return score
+
   if scorer == "default":
     return None
+  if scorer == "balanced_accuracy":
+    sk_version = tuple([int(x) for x in sklearn.__version__.split('.')])
+    if sk_version &lt; (0, 20):
```

Comment:
CDATA?

Reply:
What's CDATA? 
But here needs else clause

Reply:
https://de.wikipedia.org/wiki/CDATA

Reply:
I guess you talked about the replacement of `<` and `&` signs. `planemo lint` wants those.

Reply:
mh, unlikely, at least if you use CDATA, give me a sec

Reply:
Oh you already fixed it nice!